core/src/test/scala/org/apache/spark/status/AppStatusStoreSuite.scala

```diff
@@ -136,6 +139,51 @@ class AppStatusStoreSuite extends SparkFunSuite {
     }
   }
 
+  test("speculation summary") {
```

Comment:
Please add JIRA ID for each added test title like `SPARK-36038: speculation summary`.